core/preferences/src/commonMain/kotlin/net/thunderbird/core/preferences/GeneralSettings.kt

```diff
@@ -19,6 +19,7 @@ data class GeneralSettings(
     val fixedMessageViewTheme: Boolean,
     val isShowUnifiedInbox: Boolean,
     val isShowStarredCount: Boolean,
+    val shouldShowSetupArchiveFolderDialog: Boolean,
```

Comment:
I think you need to register this new settings also within GeneralSettingsDescriptions